tests/integration/debug_files/upload.rs

```diff
@@ -604,3 +604,130 @@ fn chunk_upload_multiple_files_small_chunks() {
         "Uploaded chunks differ from the expected chunks"
     );
 }
+
+#[test]
+/// This test is similar to `chunk_upload_multiple_files_small_chunks`, but
+/// here, only some of the chunks are missing.
+/// Of the three files two be uploaded, one is missing all chunks, and two
+/// are missing only some (including one file that is missing only one chunk).
+/// We verify that only the missing chunks get uploaded.
+fn chunk_upload_multiple_files_small_chunks_only_some() {
+    /// The chunk upload options specify that a single request should contain
+    /// at most 64 chunks.
+    const CHUNKS_PER_REQUEST: usize = 64;
+
+    /// 303 chunks will be uploaded in total.
+    const TOTAL_CHUNKS: usize = 327;
```

Comment:
Are these numbers intentionally different?

Reply:
No, that is a mistake, thanks for catching it